game/gui.py

```diff
@@ -33,16 +35,31 @@ def __init__(self, screen):
         self.load_images()
 
     def load_images(self):
-        self.graphics = {}
-        self.graphics["title"] = pygame.image.load('assets/images/title.png')
+        self.active_images = []
+        self.paused_images = []
+
+        title_image = pygame.image.load("assets/images/title.png")
+        title_image = pygame.transform.scale(title_image, (TITLE_WIDTH, TITLE_HEIGHT))
+        title_image_rect = title_image.get_rect(topleft=(
+            (BOARD_WIDTH - TITLE_WIDTH) / 2 + BOARD_X,
+            (BOARD_Y - TITLE_HEIGHT) / 2
+        ))
+        self.active_images.append((title_image, title_image_rect))
+
+        paused_image = pygame.image.load("assets/images/paused.png")
+        paused_image = pygame.transform.scale(paused_image, (PAUSED_WIDTH, PAUSED_HEIGHT))
+        title_image_rect = title_image.get_rect(topleft=(
+            (BOARD_WIDTH - PAUSED_WIDTH) / 2,
+            (BOARD_HEIGHT) / 4
+        ))
+        self.paused_images.append((paused_image, title_image_rect))
```

Comment:
Can we move the code for title image and paused image to separate methods that we call in load_images?

Reply:
How so?

As in a function for load_title_image and one for load_paused_image?

Reply:
yep